src/utils/cdn.js

```diff
@@ -1,18 +1,18 @@
-const $ = require('jquery');
-const debug = require('./debug');
+import $ from 'jquery';
+import debug from './debug.js';
 
-const CDN_ENDPOINT = process.env.CDN_ENDPOINT;
+const {CDN_ENDPOINT} = process.env;
```

Comment:
this .. may not work? iirc webpack replaces strings like `process.env.*`. have you tested this change?